runtime_tests/bun/index.test.tsx

```diff
@@ -3,6 +3,8 @@ import { serveStatic } from '../../src/adapter/bun'
 import { Context } from '../../src/context'
 import { env, getRuntimeKey } from '../../src/helper/adapter'
 import { Hono } from '../../src/index'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { jsx } from '../../src/jsx'
```

Comment:
It's not a big problem. If a user uses JSX on Bun, they can set it up correctly with `tsconfig.json`.

Reply:
I understand that this approach is acceptable, but if there is a need to modify the tsconfig.json, I am more than happy to make the necessary adjustments.

Reply:
We don't need to modify the `tsconfig.json` for this repository. Users just need to set up the `tsconfig.json` as shown below, which is already written in the starter for Bun.

```json
{
  "compilerOptions": {
    "jsx": "react-jsx",
    "jsxImportSource": "hono/jsx",
  }
}
```